docs-kits/kits/Traceability Kit/page_business_view.mdx

```diff
@@ -9,67 +9,90 @@ sidebar_position: 1
 This work is licensed under the CC-BY-4.0 (https://creativecommons.org/licenses/by/4.0/legalcode).
 
 - SPDX-License-Identifier: CC-BY-4.0
-- SPDX-FileCopyrightText: 2023 BASF SE
-- SPDX-FileCopyrightText: 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
-- SPDX-FileCopyrightText: 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST & Fraunhofer IML)
-- SPDX-FileCopyrightText: 2023 German Edge Cloud GmbH & Co. KG
-- SPDX-FileCopyrightText: 2023 Mercedes Benz AG
-- SPDX-FileCopyrightText: 2023 Robert Bosch Manufacturing Solutions GmbH
-- SPDX-FileCopyrightText: 2023 SAP SE
-- SPDX-FileCopyrightText: 2023 Siemens AG
-- SPDX-FileCopyrightText: 2023 T-Systems International GmbH
-- SPDX-FileCopyrightText: 2023 ZF Friedrichshafen AG
-- SPDX-FileCopyrightText: 2023 Contributors to the Eclipse Foundation
+- SPDX-FileCopyrightText: 2023 - 2024 BASF SE
+- SPDX-FileCopyrightText: 2023 - 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+- SPDX-FileCopyrightText: 2023 - 2024 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer ISST & Fraunhofer IML)
+- SPDX-FileCopyrightText: 2023 - 2024 German Edge Cloud GmbH & Co. KG
+- SPDX-FileCopyrightText: 2023 - 2024 Mercedes Benz AG
+- SPDX-FileCopyrightText: 2023 - 2024 Robert Bosch Manufacturing Solutions GmbH
+- SPDX-FileCopyrightText: 2023 - 2024 SAP SE
+- SPDX-FileCopyrightText: 2023 - 2024 Siemens AG
+- SPDX-FileCopyrightText: 2023 - 2024 T-Systems International GmbH
+- SPDX-FileCopyrightText: 2023 - 2024 ZF Friedrichshafen AG
+- SPDX-FileCopyrightText: 2023 - 2024 Contributors to the Eclipse Foundation
```

Comment:
Is it 2023 - 2024 or 2023, 2024 
@AngelikaWittek can you give guidance here - just to have it the same way on all pages, thanks.

Reply:
Any guidance here, @AngelikaWittek ? The [Eclipse Committer Handbook](https://www.eclipse.org/projects/handbook/#ip-copyright-headers) mentions to only use the "year in which the content was created", while the current [Open Source TRG 7.02](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) mentions a list of years.

Which approach should we follow?

Reply:
After consultation with @AngelikaWittek , please use the handbook guidelines. At the same time, we are in the process of adapting the TRG accordingly. This change refers to the representation of the year.

Reply:
Okay, I just changed the copyright year to 2023 (creation year) as mentioned in the Eclipse Committer Handbook.